robot/control/Inc/modules/LEDModule.hpp

```diff
@@ -11,6 +13,51 @@
 #include "MicroPackets.hpp"
 #include "drivers/MCP23017.hpp"
 
+
+/**
+ * Constants for DotStar LEDs (BGR)
+ * First LED in error display format: Category / Level / Info
+ */
+enum errorColors_0 : uint32_t {
```

Comment:
I would call this enum `DeviceErrorColors` or something since you're treating the three led's as a set.